lib/types/youtube.js

```diff
@@ -23,5 +23,7 @@ export const parse = ([elm]) => {
   };
 };
 
-export const render = ({youtubeId}) =>
-  <iframe src={`https://www.youtube.com/embed/${youtubeId}`} />;
+export const render = ({youtubeId, width = 640, height = 480}) => renderIframe({
```

Comment:
youtube.com uses 16:9, so I'd expect 16:9 to be default too. Probably not a big deal as it can be configured